tools/docker/Dockerfile-inclavare-dev-ubuntu18.04

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="YiLin Li <YiLin.Li@linux.alibaba.com>"
+
+RUN apt-get update && apt-get install -y make wget git \
+    libseccomp-dev gnupg gcc pkg-config libssl-dev binutils-dev debhelper \
+    libprotoc-dev protobuf-compiler libbinutils
+
+WORKDIR /root
+
+RUN mkdir /root/gopath
+
+# install prortobuf-c
+RUN wget https://github.com/protobuf-c/protobuf-c/archive/v1.3.1.tar.gz &&\
+    tar -zxvf v1.3.1.tar.gz
+
+RUN cd protobuf-c-1.3.1 &&  ./autogen.sh && \
+    ./configure && make && make install
+
+# install go
+RUN wget https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz && \
+    tar -zxvf go1.14.2.linux-amd64.tar.gz -C /usr/lib && \
+    rm -rf go1.14.2.linux-amd64.tar.gz
+
+# configure GOPATH and GOROOT
+ENV GOROOT       /usr/lib/go
+ENV GOPATH       /root/gopath
+ENV PATH         $PATH:$GOROOT/bin:$GOPATH/bin
+ENV GO111MODULE  on
+
+# install docker
+RUN apt-get install -y iptables && \
+    wget https://download.docker.com/linux/static/stable/x86_64/docker-18.09.7.tgz && \
+    tar -zxvf docker-18.09.7.tgz && mv docker/* /usr/bin
+
+# configure the rune runtime of docker
+RUN mkdir -p /etc/docker && \
+    echo "{\n\t\"runtimes\": {\n\t\t\"rune\": {\n\t\t\t\"path\": \"/usr/local/bin/rune\",\n\t\t\t\"runtimeArgs\": []\n\t\t}\n\t}\n}" >> /etc/docker/daemon.json
+
+# install SGX
+RUN echo "deb [arch=amd64] https://download.01.org/intel-sgx/sgx_repo/ubuntu bionic main" | tee /etc/apt/sources.list.d/intel-sgx.list && wget -qO - https://download.01.org/intel-sgx/sgx_repo/ubuntu/intel-sgx-deb.key | apt-key add -
+
+RUN apt-get update -y && apt-get install -y libsgx-uae-service libsgx-dcap-quote-verify libsgx-dcap-quote-verify-dev
```

Comment:
libsgx-dcap-quote-verify is dependent by libsgx-dcap-quote-verify-dev so it is not necessary to explicitly write it.

Reply:
> libsgx-dcap-quote-verify is dependent by libsgx-dcap-quote-verify-dev so it is not necessary to explicitly write it.

Ok